api/views.py

```diff
@@ -111,10 +113,10 @@ def toggleSubscriptions(request, id):
                         })
 
 #main function
-def sendAPOD(request):
+def sendAPOD():
     numbers = getNumbers()
     sendText(numbers)
-    return Response({"message": "Messages were sent successfully!"})
+    print('message sent')
```

Comment:
if `sendText()` fails, will this still print?

Reply:
addressed it

Reply:
see below